src/components/Editor/index.js

```diff
@@ -417,6 +418,25 @@ class Editor extends PureComponent {
     });
   }
 
+  onClick(e: MouseEvent) {
+    const { selectedLocation, jumpToMappedLocation } = this.props;
+
+    if (e.metaKey && e.altKey) {
+      const { line, ch } = this.state.editor.codeMirror.coordsChar({
+        left: e.clientX,
+        top: e.clientY
+      });
+
+      const sourceLocation = {
+        sourceId: selectedLocation.sourceId,
+        line: line + 1,
+        column: ch + 1
+      };
+
+      jumpToMappedLocation(sourceLocation);
```

Comment:
lets make a small util: `getSourceLocationFromMouseEvent` in `utils/editor/index.js`

```diff
from getSourceLocationFromMouseEvent(event) {
+       const { line, ch } = this.state.editor.codeMirror.coordsChar({
+         left: e.clientX,
+         top: e.clientY
+       });
+ 
+       return{
+         sourceId: selectedLocation.sourceId,
+         line: line + 1,
+         column: ch + 1
+       };
+ 
}
```